src/main/scala/cromwell/engine/backend/Backend.scala

```diff
@@ -1,35 +1,39 @@
 package cromwell.engine.backend
 
-import java.nio.file.{Files, Paths}
-
-import com.typesafe.config.Config
-import cromwell.binding
-import cromwell.binding.WdlExpression.ScopedLookupFunction
+import com.typesafe.config.{ConfigException, Config}
 import cromwell.binding._
-import cromwell.binding.types.{WdlFileType, WdlMapType}
-import cromwell.binding.values.{WdlFile, WdlMap, WdlString, WdlValue}
 import cromwell.engine.ExecutionIndex.ExecutionIndex
 import cromwell.engine._
 import cromwell.engine.backend.Backend.RestartableWorkflow
 import cromwell.engine.backend.jes.JesBackend
-import cromwell.engine.backend.local.{LocalBackendCall, LocalBackend, LocalEngineFunctions}
+import cromwell.engine.backend.local.LocalBackend
 import cromwell.engine.backend.sge.SgeBackend
 import cromwell.engine.db.DataAccess
 import cromwell.engine.workflow.CallKey
 import cromwell.parser.BackendType
 
 import scala.concurrent.{ExecutionContext, Future}
-import scala.util.{Failure, Success, Try}
+import scala.util.Try
 
 object Backend {
   class StdoutStderrException(message: String) extends RuntimeException(message)
   def from(backendConf: Config): Backend = from(backendConf.getString("backend"))
-  def from(name: String) = name.toLowerCase match {
-    case "local" => new LocalBackend
-    case "jes" => new JesBackend
-    case "sge" => new SgeBackend
-    case doh => throw new IllegalArgumentException(s"$doh is not a recognized backend")
+  def from(name: String) = try {
+    name.toLowerCase match {
+      case "local" => new LocalBackend
+      case "jes" => new JesBackend { conf } //forces configuration resolution to fail now if something is missing
+      case "sge" => new SgeBackend
+      case doh => throw new IllegalArgumentException(s"$doh is not a recognized backend")
+    }
+  } catch {
+    case e: ExceptionInInitializerError =>
+      e.getCause match {
+          // Unwrap cause if it is a configuration validation exception for more clarity
+        case c: ConfigException.ValidationFailed => throw c
+        case _ => throw e
```

Comment:
So the code above will throw a `ExceptionInInitializerError` if anything wrong happen during backend instantiation, including if it is a configuration validation failure. To make it clearer in that case that instantiation failed because of conf validation we throw the underlying `ConfigException.ValidationFailed` instead. If it's because of something else not we throw the original exception (whatever it was).